code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -1164,3 +1164,14 @@ Pass a positive integer as an argument to override a bot's default speed.
 	data["emagged"] = emagged
 	data["remote_disabled"] = remote_disabled
 	return data
+
+// AI bot access verb TGUI
+/mob/living/simple_animal/bot/proc/get_bot_data()
+	. = list(list(
```

Comment:
list containing a single other list?

Reply:
Yes. So this goes in the order
data:
    bot_type:
        specific_bot:
            get_bot_data stuff



![image](https://github.com/ParadiseSS13/Paradise/assets/66401072/e0b358f4-2f71-4a3a-80e2-585e612e8e07)

Reply:
might it be better to add it into a nested list when the value is requested? as opposed to returning a list with a list in it

Reply:
So you mean
```
data["bots"][bot_type] = list(list(bot.get_bot_data))
````
and do
```
proc/get_bot_data
. = list()
.
the rest ...
```
?

Reply:
not quite because you currently do 
`data["bots"][bot.bot_type] += bot.get_bot_data()`
so you could do 
`data["bots"][bot.bot_type] += list(bot.get_bot_data())`
and have this line be 
```	. = list(``` instead of double list ```	. = list(list(```. It's a minor change but would help make it more readable